### PR TITLE
Grammatical cleanups to doc/agent/dns

### PR DESCRIPTION
**Pull request by ryanbreen**

### CODE REVIEW

File: website/source/docs/agent/dns.html.markdown [hunk 1]
```diff
@@ -76,9 +79,8 @@ consul.			0	IN	SOA	ns.consul. postmaster.consul. 1392836399 3600 600 86400 0
 
 ## Service Lookups
 
-A service lookup is the alternate type of query. It is used to query for service
-providers and supports two lookup methods: standard lookup, and strict
-[RFC 2782](https://tools.ietf.org/html/rfc2782) lookup.
+A service lookup is is used to query for service providers.  Service queries support
```
Comment (ryanuber) [inline website/source/docs/agent/dns.html.markdown]: Double "is" here

**Comment by ryanuber** [comment]
@ryanbreen this looks great! Just one comment about a small mistake, but otherwise this looks merge-ready.

**Comment by ryanbreen** [comment]
Yep, good catch.  Bummer to make grammar worse in a PR about grammatical cleanups. :-)  Fixed.

**Comment by ryanuber** [comment]
Thanks!

**Pull request merged**